### PR TITLE
Deploy snapshots and releases

### PR DESCRIPTION
**Pull request by mateuszrzeszutek**

### CODE REVIEW

File: deploy-release.sh [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        <activeByDefault>false</activeByDefault>
+      </activation>
+      <properties>
+        <gpg.passphrase>${GPG_PASSWORD}</gpg.passphrase>
+      </properties>
+    </profile>
+  </profiles>
+</settings>
+EOF
+trap "rm release-settings.xml" EXIT
```
Comment (breedx-splk) [inline deploy-release.sh:40]: Fancy!  🐱

**Review by jkwatson — APPROVED** [review]

**Review by breedx-splk — APPROVED** [review]
Seems good to me. I like the use of `trap` here.

File: deploy-release.sh (hunk 1 above)
Comment (mateuszrzeszutek) [reply deploy-release.sh:40]: Turns out you can't set >1 trap for a single signal in bash, so I had to change that a little...

Comment (mateuszrzeszutek) [reply deploy-release.sh]: Well I ended up using `trap` after all 😅

File: pom.xml [hunk 2]
```diff
@@ -99,6 +87,19 @@
     </profile>
   </profiles>
 
+  <distributionManagement>
+    <snapshotRepository>
+      <id>ossrh</id>
```
Comment (iNikem) [inline pom.xml:92]: What is `rh`?

**Review by iNikem — APPROVED** [review]

File: pom.xml (hunk 2 above)
Comment (mateuszrzeszutek) [reply pom.xml:92]: `OSS Repository Hosting` - I copied it from Sonatype docs basically 😅

**Pull request merged**